routes/myRoute.js

```diff
@@ -0,0 +1,40 @@
+var express =require('express')    // importing express
+var app=express();                 // created object of express
+var myrouter =express.Router();
+const RegExp=require("xregexp");    // exretnal module  xregexp  to change the values 
+
+myrouter.use(express.json())
+
+
+myrouter.post("/",(req,res)=>               // router implementation
+{
+   input=req.body;   // getting the input from postman console
```

Comment:
variable x will pollute global scope. always declare variables before using.

Reply:
Ok I will do the changes